examples/main.tf

```diff
@@ -3,9 +3,26 @@ provider "kuma" {
   api_token = "test123"
 }
 
-resource "kuma_traffic_permission" "yolo_permission" {
+# resource "kuma_traffic_permission" "yolo_permission" {
+#   mesh = "default"
+#   name = "yolo_permission"
+
+#   sources {
+#     match = {
+#       "kuma.io/service" = "*"
+#     }
+#   }
+
+#   destinations {
+#     match = {
+#       "kuma.io/service" = "*"
+#     }
+#   }
+# }
+
+resource "kuma_retry" "yolo_retry" {
   mesh = "default"
-  name = "yolo_permission"
+  name = "yolo_retry"
```

Comment:
Let's name this differently 😬

Reply:
![luv-films-9DyPks6UWdofk1kCDs](https://media.giphy.com/media/84EQtaETcza8w/giphy.gif)